src/alert.js

```diff
@@ -24,7 +24,9 @@ export default class extends Controller {
   // Runs hide animation and then removes element from the page
   close() {
     leave(this.element).then(() => {
-      this.element.remove()
+      setTimeout(() => {
+        this.element.remove()
+      }, this.removeDelayValue)
```

Comment:
Good find on this. 🤦‍♂️

I believe this could be removed now? The old version wasn't using the transition library, so it was used for setting a timeout to remove.

Now that we use the transition library, we can remove it immediately after the transition since the element will be hidden.